Lib/cu2qu/rf.py

```diff
@@ -46,6 +44,103 @@ def zip(*args):
     return _zip(*args)
 
 
+class GetSegmentsPen(AbstractPen):
```

Comment:
It doesn't have to. I'd think the most likely cases for receiving a quadratic curve are: a) a user wants to pass through many fonts and have this be a no-op for fonts with quadratic curves or b) a user accidentally passes in a font with quadratic curves. Not sure if it's more useful to allow a) or catch b) (which it's currently doing).